commands/Tools/Step 3 - Modify schematisation/control_structures.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+# (c) Nelen & Schuurmans, see LICENSE.rst.
+
+import logging
+
+from PyQt4.QtCore import Qt
+
+
+from ThreeDiToolbox.views.control_structures_dockwidget import ControlStructuresDockWidget  # noqa
+from ThreeDiToolbox.commands.base.custom_command import CustomCommandBase
+from ThreeDiToolbox.utils.threedi_database import get_databases
+from ThreeDiToolbox.threedi_schema_edits.controlled_structures import \
+    ControlledStructures
+
+log = logging.getLogger(__name__)
+
+ACTIVE_TAB = " (active tab)"
+TABLE_CONTROL = 'table_control'
+BOTTOM_UP = 'Bottom up'
+RULE_OPERATOR_BOTTOM_UP = '>'
+RULE_OPERATOR_TOP_DOWN = '<'
+V2_WEIR_VIEW_TABLE = "v2_weir_view"
+V2_WEIR_TABLE = 'v2_weir'
+INITIAL_WATERLEVEL = "initial_waterlevel"
```

Comment:
At first, I thought it would be nice to have a constant, so that if it would change in the future and use on several places, it only has to be changed once. But when using the sqlalchemy ORM, that already happens in the classes. (And a better place for these constants would probably be constants.py in utils.)

(Btw, why is there also a constants.py in sql_models?)